roaringbitmap/src/main/java/org/roaringbitmap/DenseOrderedWriter.java

```diff
@@ -0,0 +1,128 @@
+package org.roaringbitmap;
```

Comment:
It's slightly unfortunate that this looks like new code in the diff.

Reply:
Yes... I actually don't know how to fix this.